manifests/m/Microsoft/OpenJDK/11/11.0.14.9/Microsoft.OpenJDK.11.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.OpenJDK.11
+PackageVersion: 11.0.14.9
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: ADDLOCAL=FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: x64
+  InstallerUrl: https://aka.ms/download-jdk/microsoft-jdk-11.0.14.9.1-windows-x64.msi#winget
+  InstallerSha256: 9932E0088AF6107297AFFEAC0560C9FE6D04DE3D041B50D1C1AABE7C8006C189
+  ProductCode: '{D2FB917E-4B7C-47AD-837D-979D31A333E1}'
+- Architecture: arm64
+  InstallerUrl: https://aka.ms/download-jdk/microsoft-jdk-11.0.14.9.1-windows-aarch64.msi#winget
```

Comment:
```suggestion
  InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/70109184-0b6e-430d-8e8e-35e8e65fe8af/bb4dceb4980d266126440c159963e8e4/microsoft-jdk-11.0.14.9.1-windows-aarch64.msi
```

Reply:
Leaving, we use the aka.ms links to monitor usage.